unity/Assets/Editor/Build.cs

```diff
@@ -51,31 +48,28 @@ private static List<string> GetAllScenePaths()
 
         foreach (string f in files) {
             if (f.EndsWith(".unity")) {
-                Debug.Log ("Adding Scene " + f);
-				scenes.Add (f);
+                Debug.Log("Adding Scene " + f);
+                scenes.Add(f);
             }
         }
 
         //uncomment for faster builds for testing
```

Comment:
also, is there any way to add a whitespace after each comment? So this becomes
```c#
// uncomment for faster builds for testing
```

Reply:
I looked at the options and couldn't find anything.